analyticsdataserver/tests/test_views.py

```diff
@@ -0,0 +1,65 @@
+import unittest.mock as mock
```

Comment:
This should be the same as the tests that were in `analyticsdataserver/tests.py`.